index.js

```diff
@@ -152,7 +152,8 @@ function jsonParseContext(serialized, reviver, opts) {
   try {
     return JSON.parse(serialized, reviver);
   } catch (e) {
-    const syntaxErr = e.message.match(/^unexpected .*position\s+(\d+)/i);
+    const syntaxErr = e.message.match(/^unexpected .*position\s+(\d+)/i)
+      || e.message.match(/^expected .*at position\s+(\d+)/i);
```

Comment:
This format supports node 20 and 22, although its not possible to run the tests against these until node18 is EOL